### PR TITLE
Use shared Data folder for client keys in integration tests

### PR DESCRIPTION
**Pull request by Rob-Hague**
And some related test changes (see the commit messages for details). These changes make my life easier in some related feature development.

### CODE REVIEW

**Comment by Rob-Hague** [comment]
@drieseng @WojciechNagorski could you kindly review/merge these test changes? It helps me with some work related to certificates.

The context is, there can be lots of combinations of keys used to create certificates. It would be cumbersome to generate and test certificates independently for the unit and integration test projects.

The other commits are to remove friction in working with the test keys with various tools

**Comment by WojciechNagorski** [comment]
I will check it.

File: test/.editorconfig [hunk 1]
```diff
@@ -28,6 +28,9 @@ dotnet_diagnostic.S1854.severity = none
 # https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0089.md
 dotnet_diagnostic.MA0089.severity = suggestion
 
+#MA0136 - Raw String contains an implicit end of line character
+dotnet_diagnostic.MA0136.severity = none
```
Comment (drieseng) [inline test/.editorconfig:32]: Use of raw string literals this raises the bar on the minimum version of C# (and therefore VS) we support.
I'm ok with this though.

Comment (Rob-Hague) [reply test/.editorconfig:32]: We already require the net7.0 SDK

**Review by drieseng — APPROVED** [review]

**Comment by Rob-Hague** [comment]
Thanks!

**Comment by WojciechNagorski** [comment]
@Rob-Hague Great job! @drieseng Thanks for review :)

**Comment by WojciechNagorski** [comment]
The 2023.0.1 version has been released to Nuget: https://www.nuget.org/packages/SSH.NET/2023.0.1

**Pull request merged**